Lib/enum.py

```diff
@@ -785,7 +785,7 @@ def __contains__(self, other):
 
     def __iter__(self):
         members, extra_flags = _decompose(self.__class__, self.value)
-        return (m for m in members if m._value_ != 0)
+        return (m for m in members if _power_of_two(m._value_))
```

Comment:
Since `int.bit_count()` was added in 3.10, I wonder if this would be more readable or give better performance:
```suggestion
        return (m for m in members if m._value_.bit_count() == 1)
```